test/relayclient/RelaySelectionManager.test.ts

```diff
@@ -57,8 +66,15 @@ contract('RelaySelectionManager', function (accounts) {
     let stubGetNextSlice: SinonStub
 
     before(async function () {
+      contractInteractor = await new ContractInteractor({
```

Comment:
howc omes you need it now, and didn't in the past?

Reply:
It used to rely on `getDependencies` method being called outside the `before` block, now there is no such method so I had to create the dependency manually.

Reply:
then we're probably missing "createDefaultDependencies()" method.

Reply:
It is not that widely needed so I think we can get away with a few more manual constructor calls...